docs/images/helpers/plugins_sdk_class_uml_diagrams

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
This file should have a extension: .puml

Reply:
Hmm. I don't think this is required for functionality. But, I can change it for making it more clear without opening the file.